documentation/docs/user_guide/inputs/pftlookup_csv.md

```diff
@@ -0,0 +1,122 @@
+# pftlookup.csv
+
+This file is required by CASA-CNP and contains biome (plant function types) specific parameters
+on soil, vegetation carbon and nutrients dynamics.
+
+| Variable       | <div style="width:250px">CABLE variable</div> | Description           |
+|----------------|-----------------------------------------------|-----------------------|
+| Kroot          | `casabiome%kroot(nv)`                         | Unused \( (m^{-1}) \) |
+| rootdepth      | `casabiome%rootdepth(nv)`                     | Unused \( (m) \) |
+| kuptake        | `casabiome%kuptake(nv)`                       | Unused \( (-) \) |
+| krootlen       | `casabiome%krootlen(nv)`                      | Unused \( (m \cdot gC^{-1}) \) |
+| KminN          | `casabiome%kminN(nv)`                         | $k_{n,up}$ an empirical parameter relating plant nitrogen uptake to soil mineral N amount. Used in `casa_nuptake()` during computation of N uptake by plants (`xnuptake`) and allocation of uptaken N to plants. \( (gN \cdot m^{-2}) \) |
+| Kuplabp        | `casabiome%Kuplabp(nv)`                       | $k_{p,up}$ an empirical parameter relating plant P uptake to labile P pool size in the soil. Used in `casa_puptake()` during calculation of P uptake by plants and the allocation to leaf wood and root. \( (gP \cdot m^{-2}) \)|
+| Fracherb       | `xfherbivore(nv)`                             | Fraction of net foliar production consumed by herbivores. Fraction of the turnover rate that affects senescence. `Fracherb=1` then senescence is only related to cold and dry stress and there is no turnover factor, if `Fracherb=0` senescence has an additional turnover rate as set by leaf age. Used to calculate `casabiome%plantrate(nv,leaf)` used in calculation of plant pool senescence rate. I think herbivores were removed when coupled to CABLE. \( (-) \) |
```

Comment:
A lot of these points are actually some notes for my own benefit. I was trying to understand what Fracherb actually is by reading the source code. Thinking: if we wet it to 0 then blah blah blah, and if we set it to 1 then blah blah blah...
As the note says, I suspect that this parameter represented herbivorous grazing in CASA standalone, but maybe it was removed? It didn't seem like a sensible implementation of grazing. I was under the impression that CABLE does not have any explicit representation of grazing. Maybe someone that knows more about CASA and CABLE coupling could shed some light on this.

Reply:
I would be astounded if we incorporated grazing!  

with leaf_age for example (there are a bunch) is it intended that the parameter in the table = /mu_leaf, and it is used as 1/mu_leaf .OR. is it expected in the table as the reciprocal already. It might be clear - I just scanned it :)

Reply:
It is a bit confusing.

- ${\mu}_{leaf}$ is the turnover rate and it's units are in ${year}^{-1}$
- $1/{\mu}_{leaf}$ is the mean residence time (a.k.a. leaf age) and it's units are in ${year}$

The table contains the latter, the mean residence time. I think CABLE internally converts these values into turnover rates by taking the reciprocal again.